lib/fizzy/types.hpp

```diff
@@ -337,6 +339,14 @@ struct Code
     // The decoded instructions' immediate values.
     // These are instruction-type dependent fixed size value in the order of instructions.
     bytes immediates;
+
+    Code() noexcept = default;
+
+    Code(uint32_t _local_count, std::vector<Instr> _instructions, bytes _immediates) noexcept
```

Comment:
Why are constructors needed again?

Reply:
A new filed is added and `Code` is used in many tests. This makes them still compile, but they don't work anyway.

> The optimization depends on `max_stack_height`. This is trivial to compute, but there are a lot of tests constructing `Code` - these require to provide this information manually. It might be good moment to upgrade these to wat2wasm tests.

So we rather need update tests rather than a `Code` constructor.

Reply:
The commit has been removed as well.